flexmeasures/api/common/schemas/sensor_data.py

```diff
@@ -344,11 +345,21 @@ def load_bdf(sensor_data: dict) -> BeliefsDataFrame:
         source = get_or_create_source(current_user)
         num_values = len(sensor_data["values"])
         event_resolution = sensor_data["duration"] / num_values
-        dt_index = pd.date_range(
-            sensor_data["start"],
-            periods=num_values,
-            freq=event_resolution,
-        )
+
+        if event_resolution == timedelta(hours=0):
+            assert (
+                num_values <= 1
+            ), "Cannot save multiple instantenous values simultaneously."
```

Comment:
Instead of raising an AssertionError, I'd rather raise a NotImplementedError. Either should result in a 500 status code.